### ISSUE TITLE
Loss function tuning

### ISSUE THREAD

**Issue by poltergeistjoya** [poltergeistjoya/Vacant_Lot_Detection#47]
Loss and IoU have been seen to diverge -- perhaps due to unideal loss weighting 

- pos_weight on BCE heavy penalization of FN on vacant pixels (lower pos weight)
- try more weight on dice than BCE
- lower pos weight in BCE -- 10 is already conservative but maybe with dice as well its not good 
- try: Lovász-Softmax loss — a direct differentiable surrogate for IoU. smp  
  has it built in. This would make loss track IoU much more closely.
